eng/liveBuilds.targets

```diff
@@ -22,6 +22,7 @@
     but we need to point to the AllArtifacts locations when building the platform manifest.
   -->
   <PropertyGroup Condition="
+    '$(RuntimeIdentifier)' != '' and
```

Comment:
I'm OK with the change, but is it expected that we have `RuntimeIdentifer` unset in some projects? 
I'm not sure why this started to be a problem only recently -- so this needs to be understood properly.
CC: @dagood

Reply:
Looking at that diff, it seems pretty straightforward. The native host project used to not care about the value of `CoreCLRArtifactsPath`. That prop has probably never had the desired value in that project in the official build, but it didn't matter because this dependency didn't exist.

https://github.com/dotnet/runtime/commit/f93065f9f05f7ebe8cf5aade4746d9d8c3154b08#diff-fa15375e5810149fbfc453d9c7d42f86

```diff
       <BuildArgs Condition="'$(CMakeArgs)' != ''">$(BuildArgs) $(CMakeArgs)</BuildArgs>
+      <BuildArgs>$(BuildArgs) -coreclrartifacts $(CoreCLRArtifactsPath)</BuildArgs>
     </PropertyGroup>
```